packages/web/src/components/nav/desktop/useNavConfig.tsx

```diff
@@ -77,26 +90,90 @@ export const useNavConfig = () => {
         leftIcon: IconFeed,
         to: FEED_PAGE,
         restriction: 'account',
-        disabled: !hasAccount
+        disabled: !hasAccount,
+        rightIcon: matchesRoute({
+          current: playingFromRoute,
+          target: FEED_PAGE
+        }) ? (
+          <IconSpeaker
+            size='s'
+            color={
+              matchesRoute({
+                current: location.pathname,
+                target: FEED_PAGE
+              })
+                ? 'staticWhite'
+                : 'accent'
+            }
+          />
+        ) : undefined
       },
       {
         label: 'Trending',
         leftIcon: IconTrending,
         to: TRENDING_PAGE,
-        restriction: 'none'
+        restriction: 'none',
+        rightIcon: matchesRoute({
```

Comment:
the config could use another config lol, works for now, gracias

Reply:
lol ikr...